### PR TITLE
[11.0][ADD] stock_move_line_po_analytic_account

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: product_move_analytical_account/hooks.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from odoo import SUPERUSER_ID, api, _
+
+
+def post_init_hook(cr, registry):
+    logging.getLogger('odoo.addons.interface_gaia').info(_('Adding Account Analytic from old purchase order line to stock.move.line'))
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+
+    for record in env["stock.move.line"].search([('account_analytic_id', '=', False)]):
+        record.account_analytic_id = record.move_id.purchase_line_id.account_analytic_id
```
Comment (dalonsod) [inline product_move_analytical_account/hooks.py]: Why not this?

```suggestion
    move_lines = env["stock.move.line"].search([('move_id.purchase_line_id.account_analytic_id', '!=', False)]):
    move_lines._compute_account_analytic()
```

Anyway, when the compute stored field is created (during addon installation) it should be automatically filled, so hook is not needed (at least in other Odoo versions it works). Could you check?

File: product_move_analytical_account/models/stock_move_line.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+from odoo import api, fields, models
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = "stock.move.line"
+
+    account_analytic_id = fields.Many2one('account.analytic.account',
+        string="Account Analytic",
+        compute="_compute_account_analytic",
+        store=True)
```
Comment (dalonsod) [inline product_move_analytical_account/models/stock_move_line.py]: Better practice:
```suggestion
    po_account_analytic_id = fields.Many2one('account.analytic.account',
        string="Analytic Account",
        compute="_compute_po_account_analytic_id",
        store=True)
```
* Prefix field name (a stock move line could belong to a sale order line, and sale order lines could apply their own analytic accounts in the future. Anyway, we can easily identify the field origin with this trick).
* Better `string` for the field in english.
* compute function should be named `_compute_<field_name>` if possible.

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments.

Also, I disagree with addon name: instead of `product_move_analytical_account` I'd prefer `stock_move_line_po_analytic_account` ("product_move" doesn't exist, and "analytical" doen't fit here). What do you think?

File: product_move_analytical_account/hooks.py (hunk 1 above)
Comment (adriresu) [reply product_move_analytical_account/hooks.py]: I tested It again and finally, is not necessary to create a hook to fill old fields, with computed store=true is enought.

File: product_move_analytical_account/models/stock_move_line.py (hunk 2 above)
Comment (adriresu) [reply product_move_analytical_account/models/stock_move_line.py]: Fixes done, and squashed commits.

**Comment by dalonsod** [comment]
Minor changes made:

- Commit reword to standard message for new addons.
- Added "PO" to analytic account label, that was present in field name definition, but not for `string` attribute.

**Comment by dalonsod** [comment]
Tested in customer test environment, installation took five seconds for more than 5,000 move lines 😄

**Pull request merged**